zipkin-server/src/main/java/zipkin/server/ZipkinServer.java

```diff
@@ -14,10 +14,11 @@
 package zipkin.server;
 
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.elasticsearch.jest.JestAutoConfiguration;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import zipkin.server.brave.BootstrapTrace;
 
-@SpringBootApplication
+@SpringBootApplication(exclude = JestAutoConfiguration.class)
```

Comment:
I'm sorry about this, but it was the only way I could find to prevent Spring from (eagerly) spinning up an unused JestClient in the application context (with pursuant confusing log messages about its initialization). I suppose we could have found a way to use that configuration, but it doesn't quite meet our needs – there's no hook for adding the signing interceptor, for example. I'd also suggest that makes more sense to have the HttpClient own its own connectivity rather than having it injected, similarly to how the NativeClient handles things.